CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## master / unreleased
 
+* [CHANGE] The frontend component now does not cache results if it finds a `Cache-Control` header and if one of its values is `no-store`
```

Comment:
Full stop and add the PR number too please :)

Reply:
And a rebase

Reply:
That was fast :smile: will do.